src/main/resources/archetype-resources/silo/src/main/resources/application.yml

```diff
@@ -45,3 +45,4 @@ remoteTokenServices:
 consul:
     name: ${spring.config.name}
     agent: 192.168.33.10
+    siloPort: ${server.port}
```

Comment:
But `port` can be misunderstood. Its the port which uses the silo as http-port not consuls port...